src/Action/LoginAction.php

```diff
@@ -52,7 +54,8 @@ public function __construct(
         TemplateRegistryInterface $templateRegistry,
         TokenStorageInterface $tokenStorage,
         TranslatorInterface $translator,
-        ?CsrfTokenManagerInterface $csrfTokenManager = null
+        ?CsrfTokenManagerInterface $csrfTokenManager = null,
+        bool $resetMail = false
```

Comment:
To be BC we should set the default value to `true`. (In case someone does `new LoginAction(...)`)

Reply:
should the default be updated in the `actions.php` too?

Reply:
yes indeed

Reply:
i made `resetMail` default to true